### DISCUSSION TITLE
Screen issue

### DISCUSSION THREAD

**Discussion by ankaa3908** [Stellarium/stellarium#2609] [Q&A] [answered]
Since I downloaded Stellarium, most windows (the large ones such as configuration, view, astronomical calculations, etc.) are not displayed correctly. Therefore, I'm not able to see the bottom of the windows (and the right part but I have to move the window) as shown in the screenshots. Is there any way I can fix this ?
Windows 10, v.0.22.2 (the screenshots were taken in full screen)
![stellarium-016](https://user-images.githubusercontent.com/111249714/187281861-35d47014-b7b8-4b8b-9b51-4b33441a6eae.png)
![stellarium-018](https://user-images.githubusercontent.com/111249714/187282109-8840338f-c011-4de3-bda7-5a73ab875e73.png)
<img width="440" alt="Capture d’écran 2022-08-30 213037" src="https://user-images.githubusercontent.com/111249714/187282976-3f3e31fd-db94-43fc-a47b-c3ad581694b4.png">

**Comment by gzotti** [comment, marked as answer]
https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#windows-gui-scaling

**Reply by ankaa3908** [reply]
Where can I access Stellarium's start menu ? I don't really understand where I can directly change the GUI scaling

**Reply by gzotti** [reply]
"In your start menu, ..." refers to the Windows Start Menu, which on Windows 10 systems most users find in the bottom left corner of the screen (unless you moved the task bar elsewhere). It usually shows the Windows logo.

**Reply by ankaa3908** [reply]
Thank you for your response but unfortunately I didn't find the "Stellarium (scale 200%)" program. Did I miss something ? 
<img width="577" alt="Capture d’écran 2022-09-01 170512" src="https://user-images.githubusercontent.com/111249714/187713015-f29f18da-c9f8-4211-b92d-4bdd72fcfe9b.png">

**Reply by alex-w** [reply]
The shortcuts "(scale XX%)" was removed from installer in version 0.22.1 or 0.22.2, but you may edit standard shortcut to enable scaling

**Reply by ankaa3908** [reply]
I can't seem to find a page in Stellarium's user guide that mentions GUI rescaling, is there somewhere else I could find a step-by-step guideline to solve this issue ? I'm not so familiar with most technical terms :/

**Reply by gzotti** [reply]
GUI rescaling is a rapidly evolving issue that seems to causes trouble just months after whichever way we try to solve it. I pointed you to the FAQ because we have not described it in the User Guide. 

Make a right-click on the "Stellarium" link in the start menu (YES, the thing that says "Stellarium" right above the word "Applications"). Say "open location". This opens an explorer window. Make a copy of the file "Stellarium.lnk". Right-click on that copy, say "Properties", and add `--scale-gui 0.75` into the command line.  Then rename the copy and give it a better name than "Copy of Stellarium.lnk".

**Reply by ankaa3908** [reply]
I did what you explained however I can't find the command line.  Stellarium 2 is the copy of Stellarium, and when I went into "properties" I searched in all the windows (respectively general, shortcuts, compatibility, security, details, and old versions) but I wasn't able to spot a line that corresponds to a command line. Yet from the description you made I'm pretty sure I'm on the right track. Sorry if fixing this problem requires so much time. 
<img width="527" alt="Screenshot 2022-09-01 224934" src="https://user-images.githubusercontent.com/111249714/187779656-819e1515-c8a7-4542-94a5-d915371dd2ad.png">
<img width="375" alt="Screenshot 2022-09-01 225019" src="https://user-images.githubusercontent.com/111249714/187779670-48f6f69b-a166-4c25-ad74-a03b874939f9.png">

**Reply by gzotti** [reply]
As soon as I click on "Properties", I see this:
![grafik](https://user-images.githubusercontent.com/18099873/187781863-39866864-cf2a-48ea-a46a-29e4d7de97ec.png)

and the line to edit is already selected.

Look into the tab "Raccourci", and edit the line possibly labeled "target". The line in my example says 

"C:\Program Files\Stellarium_1.22.3-3d47d9a\stellarium.exe" -platform windows:altgr

Make it look like 

"C:\Program Files\Stellarium_1.22.3-3d47d9a\stellarium.exe" -platform windows:altgr --scale-gui=0.75

**Reply by ankaa3908** [reply]
I just tried and weirdly nothing changes... "Cible" indeed means target so I have no clue why it's not working (and I did open Stellarium 2, not the original one) I guess I'll have to leave it as it is. 
<img width="370" alt="Screenshot 2022-09-01 235304" src="https://user-images.githubusercontent.com/111249714/187792086-fa7b442c-5f4b-4c3b-a63d-6d4cdd6748b8.png">

**Reply by ankaa3908** [reply]
Nevermind !! It worked, I just took off the =
Thank you so much for helping me :)

**Reply by gzotti** [reply]
Ah yes, no = sign, sorry. This was from my memory.
